python/ray/dashboard/modules/metrics/dashboards/train_grafana_dashboard_base.json

```diff
@@ -97,6 +97,93 @@
         "tagsQuery": "",
         "type": "query",
         "useTags": false
+      },
+      {
+        "allValue": ".+",
+        "current": {
+          "selected": false
+        },
+        "datasource": "${datasource}",
+        "definition": "query_result(count by (ray_train_run_id)(last_over_time(ray_train_worker_group_start_total_time_s{{SessionName=~\"$SessionName\",{global_filters}}}[$__range])))",
+        "description": "Filter queries to specific Ray Train run ids.",
```

Comment:
What is this? Why is this filter only for the `ray_train_worker_group_start_total_time_s`?

Reply:
This just determines how to find the values